api/HTMLLabelElement.json

```diff
@@ -70,10 +70,10 @@
               "version_added": true
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
```

Comment:
This (`control`) isn't included in https://github.com/mdn/browser-compat-data/pull/7066 but I take it that's because it really wasn't in Chrome 1. Looking forward to it in a later PR.